composer.json

```
@@ -22,7 +22,7 @@
     "require": {
         "sentry/sentry": "^2.3",
```

Due to https://github.com/getsentry/sentry-php/pull/1084#pullrequestreview-486845303:

```suggestion
        "sentry/sentry": "^2.5",
```